pkg/prowloader/testoutputmetadata.go

```diff
@@ -20,12 +21,26 @@ func GetTestOutputMetadataExtractors() map[string]TestOutputMetadataExtractorFun
 	return map[string]TestOutputMetadataExtractorFunc{
 		"Cluster upgrade.[sig-arch] Check if alerts are firing during or after upgrade success":                                                                                      alertMetadataExtractor,
 		"[sig-instrumentation][Late] Alerts shouldn't report any unexpected alerts in firing or pending state [apigroup:config.openshift.io] [Suite:openshift/conformance/parallel]": alertMetadataExtractor,
-		"[sig-arch] events should not repeat pathologically":                                                                                                                         pathologicalEventsMetadataExtractor,
-		"[sig-arch] events should not repeat pathologically in e2e namespaces":                                                                                                       pathologicalEventsMetadataExtractor,
 		"[sig-arch][Late] operators should not create watch channels very often [apigroup:config.openshift.io] [Suite:openshift/conformance/parallel]":                               watchRequestsMetadataExtractor,
 	}
 }
 
+var pathologicalTestNameMatch = regexp.MustCompile(`events should not repeat pathologically`)
+
+// testNameToMetadataExtractor takes a test name and returns an TestOutputMetadataExtractorFunc
+// via looking for the testname in the map or using a regex to match on the testName.
+func testNameToMetadataExtractor(testName string) (TestOutputMetadataExtractorFunc, error) {
+
+	extractorFunc, ok := GetTestOutputMetadataExtractors()[testName]
+	if ok {
+		return extractorFunc, nil
+	}
+	if pathologicalTestNameMatch.MatchString(testName) {
+		return pathologicalEventsMetadataExtractor, nil
+	}
+	return nil, fmt.Errorf("extractor function not found for %s", testName)
+}
+
```

Comment:
This looks ok, but could you add a test to the testoutputmetadata_test.go file and make sure we're getting what we expect? Make sure the new test output is the same as the old so we can be confident once deployed, that we're getting what we expect.

Reply:
I changed the test output to have a real log (mostly so the diff captures my new test and not part of the existing one).